packages/11ty/_includes/components/objects-catalog.webc

```diff
@@ -0,0 +1,399 @@
+  order: 3;
+}
+
+.collapsible .collapsible__panel > :first-child {
+  overflow: hidden;
+}
+
+.collapsible .collapsible__checkbox:checked ~ .collapsible__panel {
+  height: unset;
+}
+
+@media screen and (min-width: 500px) {
+  .collapsible .collapsible__checkbox:not(:checked) ~ .collapsible__panel {
+    height: 8.4em;
+  }
+}
+/* STICKY */
+
+.object-filters.sticky {
+  position: sticky;
+  z-index: 1;
+  top: 50px;
+}
+
+.sticky .object-filters__list {
+  padding-bottom: 1em;
+}
+
+/* Collapsible and sticky is not great */
+.collapsible.sticky .object-filters__list {
+  max-height: var(--max-height);
+  overflow-y: scroll;
+}
+</style>
+<section class="object-filters">
+  <input class="collapsible__checkbox" type="checkbox" id="collapsible-checkbox" />
+  <label for="collapsible-checkbox" class="collapsible__toggle"></label>
+  <div class="collapsible__panel">
+    <div class="object-filters__list">
+      <div class="object-filters__item">
+        <label for="object-type">Type:</label>
+        <div class="object-filters__select">
+          <select name="Type" id="object-type">
+            <option value="">All</option>
+            <option value="something">Something</option>
+          </select>
+        </div>
+      </div>
+      <div class="object-filters__item">
+        <label for="object-theme">Theme:</label>
+        <div class="object-filters__select">
+          <select name="Theme" id="object-theme">
+            <option value="">All</option>
+            <option value="something">Something</option>
+          </select>
+        </div>
+      </div>
+      <div class="object-filters__item">
+        <label for="object-material">Material:</label>
+        <div class="object-filters__select">
+          <select name="Material" id="object-material">
+            <option value="">All</option>
+            <option value="something">Something</option>
+          </select>
+        </div>
+      </div>
+      <div class="object-filters__item">
+        <label for="object-place">Place:</label>
+        <div class="object-filters__select">
+          <select name="Place" id="object-place">
+            <option value="">All</option>
+            <option value="something">Something</option>
+          </select>
+        </div>
+      </div>
+      <div class="object-filters__item">
+        <label for="object-period">Time Period:</label>
+        <div class="object-filters__select">
+          <select name="Time Period" id="object-period">
+            <option value="">All</option>
+            <option value="something">Something</option>
+          </select>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="collapsible__controls">
+    <div class="object-filters__controls">
+      <button class="object-filters__button object-filters__button--primary" data-control-apply>Apply Filters</button>
+      <button class="object-filters__button" data-control-reset>Reset</button>
+    </div>
+  </div>
+</section>
```

Comment:
Aligning with other single-file component frameworks it may make sense to format the WebC components with the `<template>` first, followed by setup scripts, scripts, and styles. For example:
```
<template>
</template>

<script webc:setup>
</script>

<script webc:keep>
</script>

<style webc:scoped="component">
</style>
```